src/camera/Camera.zig

```diff
@@ -0,0 +1,19 @@
+const LinuxCamera = @import("linux/LinuxCamera.zig").LinuxCamera;
+const MacOSCamera = @import("macos/MacOSCamera.zig").MacOSCamera;
+
+pub const Camera = union(enum) {
+    linux: LinuxCamera,
+    macos: MacOSCamera,
+
+    pub fn getFrame(self: Camera) void {
```

Comment:
Weird: Apparently the retun type gets ignored here. Both implementations of `getFrame()` return `[*:0]const u8` but the compiler is fine with both `void` and `[*:0]const u8` here.